ui/src/components/messages/Messages.test.tsx

```
@@ -0,0 +1,150 @@
+import { MessageProvider } from './MessageState'
+import MessagePlain from './Message'
+import MessageProgress from './MessageProgress'
+
+// Define the mock for SubscriptionsHook before using it
+const MockSubscriptionsHook = ({ messages, setMessages }: any) => {
+  return (
+    <div>
+      <button
+        onClick={() => setMessages((prev: any) => [...prev, ...messages])}
+        data-testid="trigger-messages"
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)